packages/api/src/router/admin.ts

```diff
@@ -74,4 +91,15 @@ export const adminRouter = router({
       const safetyViolations = new SafetyViolations(prisma);
       await safetyViolations.removeViolationsByRecordId(moderationId);
     }),
+  removeSafetyViolation: adminProcedure
+    .input(z.object({ recordId: z.string() }))
+    .mutation(async ({ input, ctx }) => {
+      const { recordId } = input;
+      const { prisma } = ctx;
+
+      // remove safety violation (and potentially unban user)
+      log.info("Removing safety violation", { recordId });
+      const safetyViolations = new SafetyViolations(prisma);
+      await safetyViolations.removeViolationsByRecordId(recordId);
```

Comment:
Should this return the deleted violation? Do we want logging of the intended safety violation having been deleted?

Reply:
That makes sense, I can do that 👍